src/cdk-experimental/scrolling/auto-size-virtual-scroll.ts

```diff
@@ -134,6 +134,7 @@ export class AutoSizeVirtualScrollStrategy implements VirtualScrollStrategy {
     if (this._viewport) {
       // TODO(mmalebra): Do something smarter here.
       this._setScrollOffset();
```

Comment:
Do we still want to call `_setScrollOffset`? I would think just calling `_updateRenderedContentAfterScroll` would be sufficient. Also remove TODO above

Reply:
Nope, that doesn't work. Just tried in the demo use case.

Reply:
Removed the todo and pushed.